lib/classes/Service.js

```diff
@@ -208,6 +208,28 @@ class Service {
       }
     });
 
+    const warnOnDuplicateHandlers = () => {
+      const functionObjs = this.getAllFunctions()
+        .map(name => ({
+          name,
+          body: this.getFunction(name),
+        }))
+        .filter(func => _.isString(func.body.handler));
+      const groupedByHandler = _.groupBy(functionObjs, func => func.body.handler);
+      _.entriesIn(groupedByHandler)
+        .filter(arg => arg[1].length >= 2)
+        .forEach(arg => {
+          const handlerName = arg[0];
+          const functionGroup = arg[1].map(func => func.name);
+          const warnMessage = [
+            `A handler "${handlerName}" is duplicated in functions: `,
```

Comment:
Please add `Warning: ` prefix like this
<img width="618" alt="2018-12-31 15 09 31" src="https://user-images.githubusercontent.com/1301012/50555357-61060b00-0d0e-11e9-9a64-66f2c2b020af.png">

Reply:
Done in https://github.com/serverless/serverless/pull/5638/commits/5df4c7982d8e59cf9e4342a158c8f7d0f173d5a3